### PR TITLE
Fixes #84 and #83 : Fixed some design issues.

### LINKED ISSUE #83: Properly align Content section and use have a larger wrapper.
**I am working on it.**

### LINKED ISSUE #84: Change links to be of color FOSSASIA red.
**Working on it!**

### PR DESCRIPTION
**Pull request by nzec**
* Properly aligned the top buttons.
* Made the partners background white.
* Made the Our Friends section with design similar to other FOSSASIA pages.
* Center aligned the content section.
* Fixed some minor indentation.
* Fixes #84 
* Fixes #83 

@0x48piraj collaborated with me on #83 #84
***Preview Link : https://duttaditya18.github.io/2014.fossasia.org/***

### CODE REVIEW

**Review by realslimshanky — APPROVED** [review]
LGTM

File: _includes/header.html [hunk 1]
```diff
@@ -15,16 +15,16 @@
...
+			letter-spacing: 3px;
+			text-align: center;
 		}
 
 		.partners .sponsor {
 			margin: 0 auto;
 			margin-top: 10%;
 			padding-bottom: 3%;
-            		width: 200px;
+    		width: 200px;
```
Comment (Ploypiti) [inline _includes/header.html]: Please fix an indentation.

Comment (nzec) [reply _includes/header.html]: Fixed!

**Review by Ploypiti — APPROVED** [review]
LGTM 🐍

**Review by yashLadha — APPROVED** [review]
:+1: LGTM

**Pull request merged**